features/step_definitions/government_steps.rb

```diff
@@ -20,3 +20,7 @@
     end_date: end_date
   })
 end
+
+Given(/^there is a current government$/) do
+  FactoryGirl.create(:current_government)
```

Comment:
This is true, though most other apps are explicit.  David and I also prefer more explicitness in general, though if you're after a specific reason, being able to easily tell apart FactoryGirl helpers vs the custom ones seems like a benefit.